vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator_test.go

```diff
@@ -410,3 +413,46 @@ func TestNoRecommendationForContainer(t *testing.T) {
 	result := calculator.getUpdatePriority(pod, nil)
 	assert.NotNil(t, result)
 }
+
+func TestGetUpdatePriority_VpaObservedContainers(t *testing.T) {
+	tests := []struct {
+		name           string
+		pod            *apiv1.Pod
+		recommendation *vpa_types.RecommendedPodResources
+		want           float64
+	}{
+		{
+			name:           "with no VpaObservedContainers annotation",
+			pod:            test.Pod().WithName("POD1").AddContainer(test.BuildTestContainer(containerName, "1", "")).Get(),
+			recommendation: test.Recommendation().WithContainer(containerName).WithTarget("10", "").Get(),
+			want:           9,
```

Comment:
The 9 and 0 seem a bit cryptic here, it's not clear what we are actually expecting. If test fails because it should have been 0 but it is 9, what does that mean?
 Can you change test case name to include expectation or add a comment?

Reply:
Comment and consts added.

Reply:
Thanks! Reads way better now.